bin/jetstream.php

```diff
@@ -24,4 +24,7 @@
 
 $app = \Friendica\App::fromDice($dice);
 
-$app->processJetstream();
+$argv = $_SERVER['argv'] ?? [];
```

Comment:
Can we add a deprecation note here, so the user who run `bin/jetstream.php` will see something like `bin/jetstream.php is deprecated and will removed. Please use bin/console.php jetstream instead`.